src/frontend/screens/Onboarding/JoinProjectQrScreen.tsx

```diff
@@ -48,18 +76,34 @@ export const JoinProjectQrScreen: NavigationStackScreenComponent = () => {
               <FormattedMessage {...m.instructionsTitle} />
             </Text>
             <Text style={styles.instructionsDescription}>
-              <FormattedMessage {...m.instructionsDescription} />
+              {isAdmin ? (
+                <FormattedMessage {...m.instructionDescriptionAsAdmin} />
+              ) : (
+                <FormattedMessage {...m.instructionsDescription} />
+              )}
             </Text>
           </View>
         </View>
-        <Button
-          variant="text"
-          onPress={() => navigation.navigate("SendJoinRequest")}
-        >
-          <Text style={styles.sendJoinRequest}>
-            <FormattedMessage {...m.sendJoinRequest} />
-          </Text>
-        </Button>
+
+        {isAdmin ? (
+          //Admin cancel Button
+          <Button
+            variant="outlined"
+            onPress={() => navigation.dispatch(resetAction)}
+          >
+            {t(m.cancel)}
```

Comment:
is there a reason we can't use the `<Text ...><FormattedMessage ... /></Text>` style here?

Reply:
This jsut avoids having to style the text. By using a string directly, the button styling automatically applies to the text

Reply:
oh interesting, good to know!